doc/docs/Python_Tutorials/Cylindrical_Coordinates.md

```diff
@@ -455,6 +455,156 @@ As shown below, the results for the scattering cross section computed using cyli
+                mp.Source(mp.GaussianSource(frq_cen, fwidth=dfrq),component=mp.Ep, center=src_center_side,size=mp.Vector3(z=src_size_side), amplitude = Jside, amp_func = amp_side),
+                ]
+
+
+            sim = mp.Simulation(
+                cell_size=cell_size,
+                boundary_layers=pml_layers,
+                resolution=resolution,
+                sources=sourcesp,
+                dimensions=mp.CYLINDRICAL,
+                m=cur_m,
+                force_complex_fields = True,
+                accurate_fields_near_cylorigin=True,
+                Courant=min(0.5, 1/(abs(cur_m)+0.5)))
+
+            box_z1 = sim.add_flux(frq_cen, dfrq, nfrq,
+                mp.FluxRegion(center=mp.Vector3(0.5 * r, 0, -0.5 * h), size=mp.Vector3(r)))
+            box_z2 = sim.add_flux(frq_cen, dfrq, nfrq,
+                mp.FluxRegion(center=mp.Vector3(0.5 * r, 0, +0.5 * h), size=mp.Vector3(r)))
+            box_r = sim.add_flux(frq_cen, dfrq, nfrq,
+                mp.FluxRegion(center=mp.Vector3(r), size=mp.Vector3(z=h)))
+
+
+            sim.run(until_after_sources=10)
+
+            freqs = mp.get_flux_freqs(box_z1)
+            box_z1_data = sim.get_flux_data(box_z1)
+            box_z2_data = sim.get_flux_data(box_z2)
+            box_r_data = sim.get_flux_data(box_r)
+            box_z1_flux0 = mp.get_fluxes(box_z1)
+
+
+            sim.reset_meep()
+
+            sim = mp.Simulation(
+                cell_size=cell_size,
+                geometry=geometry,
+                boundary_layers=pml_layers,
+                resolution=resolution,
+                sources=sourcesp,
+                dimensions=mp.CYLINDRICAL,
+                m=cur_m,
+                force_complex_fields = True,
+                accurate_fields_near_cylorigin=True,
+                Courant=min(0.5, 1/(abs(cur_m)+0.5)))
+
+            box_z1 = sim.add_flux(frq_cen, dfrq, nfrq,
+                mp.FluxRegion(center=mp.Vector3(0.5 * r, 0, -0.5 * h), size=mp.Vector3(r)))
+            box_z2 = sim.add_flux(frq_cen, dfrq, nfrq,
+                mp.FluxRegion(center=mp.Vector3(0.5 * r, 0, +0.5 * h), size=mp.Vector3(r)))
+            box_r = sim.add_flux(frq_cen, dfrq, nfrq,
+                mp.FluxRegion(center=mp.Vector3(r), size=mp.Vector3(z=h)))
+
+
+            sim.load_minus_flux_data(box_z1, box_z1_data)
+            sim.load_minus_flux_data(box_z2, box_z2_data)
+            sim.load_minus_flux_data(box_r, box_r_data)
+
+
+            sim.run(until_after_sources=100)
+
+            box_z1_flux = mp.get_fluxes(box_z1)
+            box_z2_flux = mp.get_fluxes(box_z2)
+            box_r_flux = mp.get_fluxes(box_r)
+
+            scatt_flux_m[alpha_i, cur_m] = box_z1_flux[0] - box_z2_flux[0] - box_r_flux[0]
+            sim.reset_meep()
+
+scatt_power_m = np.zeros((alpha_range, mrange+1))
+for i in range(mrange+1):
+  scatt_power_m[:,i] = - 2*np.sum(scatt_flux_m[:,0:(i+1)], axis=1) + scatt_flux_m[:,0]
```

Comment:
The two-space indentation is inconsistent with the four-space indentation in the rest of the script. This will produce a runtime error. Please fix.